src/sentry/static/sentry/app/components/u2f/u2finterface.tsx

```diff
@@ -6,7 +6,29 @@ import * as Sentry from '@sentry/react';
 import ConfigStore from 'app/stores/configStore';
 import {t, tct} from 'app/locale';
 
-class U2fInterface extends React.Component {
+type ChallengeData = {
+  authenticateRequests: u2f.SignRequest;
+  registerRequests: u2f.RegisterRequest;
+};
+
+type Props = {
+  challengeData: ChallengeData;
+  flowMode: string;
+  style?: React.CSSProperties;
+  silentIfUnsupported: boolean;
+  onTap: ({response, challenge}) => Promise<any>;
```

Comment:
Is there a better return type than `any` here?

Reply:
I didn't see any returns in the code, so undefined I guess.